src/ClientData/FunctionInfo.cpp

```diff
@@ -50,7 +50,18 @@ std::optional<uint64_t> FunctionInfo::GetAbsoluteAddress(const ProcessData& proc
 
 bool FunctionInfo::IsFunctionSelectable() const {
   constexpr const char* kLibOrbitUserSpaceInstrumentation = "liborbituserspaceinstrumentation.so";
-  return module_path().find(kLibOrbitUserSpaceInstrumentation) == std::string::npos;
+  if (module_path().find(kLibOrbitUserSpaceInstrumentation) != std::string::npos) {
+    return false;
+  }
+
+  constexpr const char* kNameOfWineSyscallDispatcher = "__wine_syscall_dispatcher";
+  constexpr const char* kNameOfWineSyscallDispatcherModule = "ntdll.so";
+  if (pretty_name().find(kNameOfWineSyscallDispatcher) != std::string::npos &&
```

Comment:
Nit: Consider using `absl::StrContains` for slightly better readability.

Reply:
Yep, thanks, I prefer that as well.